### DISCUSSION TITLE
Missing Sensor values from HC_Meters

### DISCUSSION THREAD

**Discussion by savage7** [Arnold-n/P1P2MQTT#141] [Q&A] [answered]
Do I need to enable anything to get sensor reading values for theses Sensor values in HC_Meters?
I'm already published to P1P2/P/meter/U/9/Electricity_Power which gives me nice values for Realtime COP. But these don't seam to work.

<img width="313" alt="Bildschirmfoto 2024-11-19 um 14 11 21" src="https://github.com/user-attachments/assets/738ab6aa-ad5c-427d-bcf3-8358770d7149">
<img width="299" alt="Bildschirmfoto 2024-11-19 um 14 11 06" src="https://github.com/user-attachments/assets/6afff875-7462-4d98-9ae1-eb54af442ff9">

**Comment by Arnold-n** [comment, marked as answer]
Do you have Counter_Request_Function (command `C2`) enabled?
![Screenshot 2024-11-19 at 14-35-24 HC – Home Assistant](https://github.com/user-attachments/assets/63bf0182-e50e-495e-8965-4d4d0f92a8a9)

**Comment by savage7** [comment]
@Arnold-n thanks alot for your super fast help 🥇
